### PR TITLE
Initial commit for  mta v6.2.2 and v7.0.2 manifests

### PR DESCRIPTION
**Pull request by rhkp**
Hi @dmartinol / @masayag ,
Please review and merge or provide feedback. 

This is part 2 (instructions 5-7) as provided at https://github.com/parodos-dev/serverless-workflows?tab=readme-ov-file#to-introduce-a-new-workflow

Thanks.

### CODE REVIEW

**Comment by dmartinol** [comment]
it's a pity that we have a lot of duplicated code, but probably acceptable in this context. Maube Moti and Gabriel have a different opinion on this topic.
maybe the `mta` folder is useless and can be removed?
I would also remove all the initial manifests generated by the `serverless-workflows` automation, because they miss the additional manipulation performed by the GH action (e.g., setting prod profile, adding persistence and image configuration)

**Comment by masayag** [comment]
@rhkp is this PR rebased on the workflows branch? Can we adjust the versions to match the one from the origin repository?

**Comment by rhkp** [comment]
> @rhkp is this PR rebased on the workflows branch? Can we adjust the versions to match the one from the origin repository?

Sure. Let me do that. Thanks.

**Comment by rhkp** [comment]
@masayag Thanks for the comment. 
Submitting the updated manifests based on changes in underlying MTA workflows including changes to the dir names.
Please let me know of any changes that you may wish me to do or approve.

**Review by masayag — APPROVED** [review]

**Review by masayag — CHANGES_REQUESTED** [review]
Sorry for missing this part, but I can't see the container image built for those workflows from the serverless-workflows repository specified in the sonataflow CRs.

File: charts/mtav6.x/templates/01-sonataflow_mta-analysis-v6.yaml [hunk 1]
```diff
@@ -0,0 +1,537 @@
...
+      end:
+        terminate: true
+      name: NotifyFailureBackstage
+      type: operation
+    timeouts:
+      workflowExecTimeout:
+        duration: PT1H
+  podTemplate:
+    container:
+      resources: {}
```
Comment (masayag) [inline charts/mtav6.x/templates/01-sonataflow_mta-analysis-v6.yaml:526]: the image should be specified here, in a similar way as done for the existing mta workflow:
https://github.com/parodos-dev/serverless-workflows-config/blob/main/charts/mta/templates/01-sonataflow_mtaanalysis.yaml#L252

so it appears the repository was never created and the image wasn't pushed: https://github.com/parodos-dev/serverless-workflows/actions/workflows/mta-v6.x.yml

I'm going to create the repositories in quay for both workflows (v6 and v7) and repeat the failed actions.
Only then this CR can be updated.

Comment (masayag) [reply charts/mtav6.x/templates/01-sonataflow_mta-analysis-v6.yaml:526]: now things looks better - there is an image created successfully at `quay.io/orchestrator/serverless-workflow-mta-v6.x:1bde42520bd287ded1eab55380a2d17912f0f32a`
and it needs to be added here.

then, by merging this PR, the next failed task in the job below should be resolved:
 https://github.com/parodos-dev/serverless-workflows/actions/runs/9552838333/job/26330499375

Comment (masayag) [reply charts/mtav6.x/templates/01-sonataflow_mta-analysis-v6.yaml:526]: in the meantime, running the build for mta-v7

**Review by gabriel-farache — CHANGES_REQUESTED** [review]
Remove the 01-XX 02-XX and 03-XX files, they will be generated by the CI

File: charts/mtav7.x/templates/03-configmap_mta-analysis-v7-props.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+data:
+  application.properties: |
+    quarkus.kogito.devservices.enabled=false
+
+    mta.url = ${MTA_URL:http://tackle-ui.my-konveyor-operator.svc.cluster.local:8080}
+    quarkus.rest-client.mta_json.url = ${mta.url}
+    quarkus.rest-client.notifications.url=${BACKSTAGE_NOTIFICATIONS_URL:http://backstage-backstage.rhdh-operator/api/notifications/}
+    quarkus.openapi-generator.mta_json.auth.bearerAuth.bearer-token=${MTA_HUB_TOKEN}
```
Comment (gabriel-farache) [inline charts/mtav7.x/templates/03-configmap_mta-analysis-v7-props.yaml:9]: What is this and how to generate one?
There is no such props in https://github.com/parodos-dev/serverless-workflows/blob/main/mta-v7.x/application.properties

Comment (rhkp) [reply charts/mtav7.x/templates/03-configmap_mta-analysis-v7-props.yaml:9]: These were generated using make command with gen-manifests option.

https://github.com/parodos-dev/serverless-workflows/blob/main/Makefile#L173

**Review by masayag — APPROVED** [review]

**Pull request merged**